content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -186,6 +177,39 @@ For example:
    }
    ```
 
+### YAML resource encoding {#yaml-encoding}
+Kubernetes also supports the [`application/yaml`](https://www.rfc-editor.org/rfc/rfc9512.html) media type for both requests and responses. [`YAML`](https://yaml.org/) can be used for defining Kubernetes manifests and API interactions.
+
+For example:
+
+1. List all of the pods on a cluster in YAML format
+   ```
+   GET /api/v1/pods
+   Accept: application/yaml
+   ---
```

Comment:
nit, `---` is confusing when documenting yaml examples since it is actually a yaml separator... consider breaking the request and response into distinct code blocks instead (same comment below)

Reply:
I agree, but the docs are much better with this change, and obviously buggy without it.